docs/topics/time.md

```diff
@@ -0,0 +1,373 @@
+[//]: # (title: Time)
+
+The Kotlin standard library gives you the tools you need to be able to calculate and measure time in different units.
+Being able to measure time accurately is important for activities like:
+  * Managing threads or processes
+  * Collecting statistics
+  * Detecting timeouts
+  * Debugging
+
+By default, time is measured using a monotonic time source but other time sources can be configured.
+
+## Durations
+
+To represent an amount of time, the standard library has the [`Duration`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.time/-duration/)
+class. A `Duration` can be expressed in the following units from the [`DurationUnit`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.time/-duration-unit/)
+Enum class:
+  * `NANOSECONDS`
+  * `MICROSECONDS`
+  * `MILLISECONDS`
+  * `SECONDS`
+  * `MINUTES`
+  * `HOURS`
+  * `DAYS`
+
+`Durations` can also be zero, negative or infinite.
+
+### Calculate duration
+
+To create an instance of the `Duration` class, use the [extension properties](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.time/-duration/#companion-object-properties) 
```

Comment:
Duration is a value class and therefore it doesn't have an instance identity. Nothing is created (in sense of object creation) in the result of these calls.

Reply:
I updated the text. Is it OK now?